website/src/releases/2023-05/new-theme.mdx

```diff
@@ -0,0 +1,18 @@
+---
+date: 2023-05-09
+title: New UI styles for Merchant Center
+description: The Merchant Center styles have been updated.
+type: announcement
+topics:
+  - Merchant Center
+  - Development
+---
+
+Now that we've redesigned the Merchant Center to enhance user experience and usability, you can also make your Custom Application look consistent with the rest of the Merchant Center.
+You'll just need to update your `ui-kit` and `app-kit` dependencies (use the latest version if possible).
```

Comment:
@emmenko My main concern here is that we're not telling developers which version they should use for their Custom Applications to look like the rest of the Merchant Center.
If I was such a developer, I would be wondering: "Ok, if I don't use the latest versions, what are the minimum versions I should use to take full advantage of the new design?"

Reply:
Updated here: f7e78ed415646fc7393db468b61859a8790fc0ee